codecov/settings_base.py

```diff
@@ -217,6 +231,7 @@
         "HOST": DATABASE_READ_HOST,
         "PORT": DATABASE_READ_PORT,
         "CONN_MAX_AGE": CONN_MAX_AGE,
+        "OPTIONS": ast.literal_eval(DATABASE_READ_OPTIONS),
```

Comment:
Could this be a `json.loads` instead here instead?

Reply:
Sure - would then be a more restricted format, though.

Reply:
In what sense?  Were you hoping to pass in Python literals that can't be encoded as JSON?

Reply:
> Were you hoping to pass in Python literals that can't be encoded as JSON?

Not really. I think I can life with JSON just fine.